pkg/operator/ceph/nfs/nfs.go

```diff
@@ -48,41 +53,51 @@ func (c *CephNFSController) upCephNFS(n cephv1.CephNFS, oldActive int) error {
 		n.Spec.Server.Active-1)
 
 	for i := oldActive; i < n.Spec.Server.Active; i++ {
-		name := k8sutil.IndexToName(i)
+		id := k8sutil.IndexToName(i)
 
-		configName, err := c.generateConfig(n, name)
+		configName, err := c.generateConfig(n, id)
 		if err != nil {
 			return fmt.Errorf("failed to create config. %+v", err)
 		}
 
-		err = c.addRADOSConfigFile(n, name)
+		err = c.addRADOSConfigFile(n, id)
 		if err != nil {
 			return fmt.Errorf("failed to create RADOS config object. %+v", err)
 		}
 
+		cfg := daemonConfig{
+			ID:              id,
+			ConfigConfigMap: configName,
+			DataPathMap: &config.DataPathMap{
```

Comment:
I was considering that, but it would have to be `NewLoglessDataPathMap`, and we presumably might want a `NewDatalessLoglessDataPathMap`, and doing all permutations of these sounds a little too much. I'm inclined to have `New...DataPathMap` functions for things that do log so that there is ensured consistency around the logging dir, but here `config.DataPathMap` is well-documented enough that I think using it directly makes the most sense.

Reply:
Sounds reasonable,  I am not going to dwell on that too much.